.editorconfig

```diff
@@ -13,3 +13,6 @@ trim_trailing_whitespace = false
 
 [*.json]
 indent_style = space
+
```

Comment:
If you can get .prettierrc to work as I suggested on flowdock maybe you could remove this commit. I agree that we should improve the module configuration so someone creating their first PR doesn't get caught in formatting issues. Maybe adding prettify to `lint-staged` @20k-ultra ?

Reply:
Yeah, the suggestion of adding `.prettierrc` is gold, since `.editorconfig` doesn't actually tell the IDE to format things in the same way as is done when running `prettify`.  If it were included in the README under development, or just as a footnote in a potential "Intro to Supervisor" Q&A gist, that would be great.

Reply:
@pipex  Agreed, will remove this commit pending comment from @20k-ultra (since I'm not sure if comments on deleted commits remain in GH -- does anyone know?)

Reply:
Not sure, but you can remove the commit if you want, we can always take the conversation back to FD

Reply:
Comments stay, apparently. Commit removed 👍🏼